crates/detect-wasi/Cargo.toml

```
@@ -0,0 +1,13 @@
+[package]
+name = "detect-wasi"
+description = "Detect if WASI can be run"
+repository = "https://github.com/cargo-bins/cargo-binstall"
+documentation = "https://docs.rs/detect-wasi"
+version = "1.0.0"
```

I would prefer `0.1.0`:

```suggestion
version = "0.1.0"
```

The name `detect-wasi` is available.

---

This trivial crate in particular is basically done, so I'm fine calling it 1.0.

---

> This trivial crate in particular is basically done, so I'm fine calling it 1.0.

hmmm, that sounds ok.